TheOracle2/Interactions/SlashCommands/ProgressTrackCommand.cs

```diff
@@ -0,0 +1,116 @@
+﻿using Discord.Interactions;
```

Comment:
this being a single command has the same rationale as `/clock`, above

name change rationale: `progress-track` is a little more descriptive than `track` (which is also a verb). it also means that someone who starts typing `/progress` (which i suspect is the first thing many people will try) will see this command.